src/utils/useMouseTracking.js

```diff
@@ -51,21 +74,17 @@ export const useMouseTracking = () => {
 				if (mouseTrackingActive && data.pageX != undefined && data.pageY != undefined) {
 					deviceRef.current = await getUserDevice();
 
-					if (!batchReferenceRef.current) {
-						batchReferenceRef.current = doc(
-							db,
-							`users/${user.uid}/visitedPages/${data.currentWebPageTitle}/mouseTracking`,
-							'mouseData',
-						);
-					}
+					batchReferenceRef.current = doc(
+						db,
+						`users/${user.uid}/visitedPages/${data.currentWebPageTitle}/mouseTracking`,
+						'mouseData',
+					);
```

Comment:
Ik bedenk me net dat dit niet in een useRef moet zitten. Er mag toch elke keer zo een reference aangemaakt worden? Dus gewoon elke keer `const batchReference = doc(....)`